frontend/src/components/FormControl/FormLabel/FormLabel.tsx

```diff
@@ -145,7 +148,12 @@ const FormLabelDescription = ({
   })
 
   return useMarkdown ? (
-    <ReactMarkdown components={mdComponents}>{children}</ReactMarkdown>
+    // Wrap markdown with a <div white-space='pre-wrap'> to get consecutive newlines to show up
+    <Box whiteSpace="pre-wrap">
+      <ReactMarkdown components={mdComponents} remarkPlugins={[gfm]}>
```

Comment:
as discussed, should we create a separate `TextMarkdown` component that uses all the plugins we want to have a consistent markdown usage across the app?

also use something like [remark-breaks](https://github.com/remarkjs/remark-breaks) so breaks are (in)correctly rendered. (breaks markdown spec but for our use case its fine lol)

Reply:
looking at https://github.com/vmware-tanzu/kubeapps/pull/4765 but it didn't work - some error with gfm, breaks and react-markdown not playing nice together.

Tried following this "little plugin" but it's still not working (now it shows up worse than it would have with pre-wrap): https://github.com/remarkjs/react-markdown/issues/278#issuecomment-971895356

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/25571626/195064054-1b961bd3-9cf6-4a5a-a582-1845f9702630.png">